addOns/openapi/openapi.gradle.kts

```diff
@@ -82,7 +82,7 @@ dependencies {
         exclude(group = "com.github.java-json-tools", module = "json-schema-validator")
         exclude(group = "org.apache.httpcomponents", module = "httpclient")
     }
-    implementation("org.apache.commons:commons-lang3:3.19.0")
+    implementation(libs.openapi.apache.commons.lang3)
```

Comment:
I'd keep this one here, this is more an internal detail (it should also be dropped when targeting newer ZAP core).